### ISSUE TITLE
Silence detection is too sensitive

### ISSUE THREAD

**Issue by spameier** [radiorabe/klangbecken#83]
The silence detection in [klangbecken.py](https://github.com/radiorabe/klangbecken/blob/d4f0d8117b8f31e218e538a9128bff4a21a2fd04/klangbecken.py#L171) is with 0.01 seconds of silence very low and aggressive. We should consider changing it to maybe 0.1 seconds.

Example: [Rhythm Is Love - Keziah Jones on 03.12.20 13:14](https://archiv.rabe.ch/2020/12/03;play=mid;format=mp3;time=131419) (Login required)

**Comment by smlz** [comment]
ffmpeg silencedetect works in mysterious ways. I tried a lot of values for the duration to generate reliable results, though there are probably better values than 0.01.

I tried to "fix" the cases, when unrealistically large cue_in values were generated (https://github.com/radiorabe/klangbecken/blob/d4f0d8117b8f31e218e538a9128bff4a21a2fd04/klangbecken.py#L204), but this was not enough, as we see now.

The "fix" in the PR simply ignores all cue_in values greater than 5 seconds.

(The code also checks for negative cue in values, and I had to append silence to the end of the files, to make the calculation of cue out points reliable. There is some test code to verify the correct calculation of cue points: https://github.com/radiorabe/klangbecken/blob/d4f0d8117b8f31e218e538a9128bff4a21a2fd04/tests/test_api.py#L445-L465)

For now I think that this is a sufficiently reliable "fix", but we probably should think about switching to gstreamer (#20).
